backend/sse-edge-api/src/main/resources/sse-api.yaml

```diff
@@ -4,6 +4,36 @@ info:
   description: This describes the API clients use for accessing features
   version: "1.1.1"
 paths:
+  /features/:
+    get:
+      tags:
+        - FeatureService
+      parameters:
+        - name: sdkUrl
```

Comment:
are we certain that sdkUrl is the right name? not something like serviceAccountUrl ?

Reply:
or  envServiceAccountUrl or envSdkUrl?

Reply:
its an sdkUrl because it is the cache-name + the api key for the service account + the environment id. I don't want it to be longer, because, well, data and we use sdkUrl everywhere else now...

Reply:
i am not convinced the name makes sense, but perhaps it's a larger change as like you say it is used in other places